quic/s2n-quic-transport/src/ack/ack_manager.rs

```diff
@@ -633,7 +633,10 @@ mod tests {
 
     #[test]
     fn size_of_snapshots() {
+        #[cfg(target_pointer_width = "64")]
         assert_debug_snapshot!("AckManager", size_of::<AckManager>());
+        #[cfg(target_pointer_width = "32")]
+        assert_debug_snapshot!("AckManager32", size_of::<AckManager>());
```

Comment:
I think it would be better to just gate these tests on 64 bit platforms:

```suggestion
    #[cfg_attr(not(target_pointer_width = "64"), ignore)]
    fn size_of_snapshots() {
        assert_debug_snapshot!("AckManager", size_of::<AckManager>());
```